microservices/embeddingservice/src/embed_text.py

```diff
@@ -78,10 +78,10 @@ def load_all_models(self):
 
         for key, configuration in self.config[MODELS_KEY].items():
             if C2C_MODELS_KEY in configuration:
-                self.model_configs = (configuration[C2C_MODELS_KEY])
+                self.model_configs.update(configuration[C2C_MODELS_KEY])
 
         for model, model_config in self.model_configs.items():
-            if model_config["endpoint"].startswith("opensearch://"):
+            if "model_path" in model_config and model_config["model_path"] not in [None, ""]:
```

Comment:
```suggestion
            if not model_config.get("model_path"):
                continue
```

all builtin data structures have an inherent truthy value. None and "" both evaluate to False. 
Also we include the key check by simply getting the value with None as default.

Also while we are at it I would prefer if we invert the statement treat it as a guard clause which helps readability and reduces one level of indentation.